services/catalog/rs_server_catalog/user_catalog.py

```diff
@@ -567,6 +567,21 @@ def update_links_for_all_collections(self, collections: list[dict]) -> list[dict
                 link["href"] = link_parser._replace(path=new_path).geturl()
         return collections
 
+    def update_stac_catalog_metadata(self, metadata: dict):
+        """Update the metadata fields from a catalog, following the US RSPY-282
+
+        Args:
+            metadata (dict): The metdata that has to be updated. The fields id, title,
+                            description and stac_version are to be updated, by using the env vars which have
+                            to be set before starting the app/pod. The existing values are used if
+                            the env vars are not found
+        """
+        if metadata.get("type") != "Catalog":
```

Comment:
Why not just

`python
if metadata.get("type") == "Catalog":
   for ..
`

Reply:
fixed (personal point of view)